src/browser-frontend/selectors/ui-pages-selectors.js

```diff
@@ -44,8 +44,16 @@ export function getPageTabState(state, pageId) {
   return getPageVisuals(state, pageId).get('tabState');
 }
 
+export function getOptionalClass(state, pageId) {
+  return getPageVisuals(state, pageId).get('optionalClass');
+}
+
 // UI page computed properties getters.
 
+export function getPageDisplayOwner(state, pageId) {
+  return !!PagesSelectors.getPageOwnerId(state, pageId);
```

Comment:
I see. Separating these concerns makes a ton of sense for the model, but selectors don't really matter as much IMO. But feel free to make these distinctions if you think it's better.